core/client-internal/src/main/java/tachyon/client/block/TachyonBlockStore.java

```diff
@@ -96,11 +95,8 @@ public BlockInfo getInfo(long blockId) throws IOException {
     BlockMasterClient masterClient = mContext.acquireMasterClient();
     try {
       for (WorkerInfo workerInfo : masterClient.getWorkerInfoList()) {
-        NetAddress address = new NetAddress(workerInfo.getAddress().getHost(),
-            workerInfo.getAddress().getRpcPort(), workerInfo.getAddress().getDataPort(),
-            workerInfo.getAddress().getWebPort());
-        infoList.add(
-            new BlockWorkerInfo(address, workerInfo.getCapacityBytes(), workerInfo.getUsedBytes()));
+        infoList.add(new BlockWorkerInfo(workerInfo.getAddress(), workerInfo.getCapacityBytes(),
```

Comment:
High level question: what do you think of isolating all the thrift to wire structure conversion in the client? So that all the thrift structures are only used by thrift client. This also prepares for future RPC migration, such as protobuffer or other protocol.